samcli/commands/list/cli_common/options.py

```diff
@@ -0,0 +1,33 @@
+"""
+Common CLI options shared by various commands
+"""
+
+import click
+
+
+def stack_name_click_option():
```

Comment:
We could probably wrap these common options in a single function like https://github.com/aws/aws-sam-cli/blob/develop/samcli/commands/local/cli_common/options.py#L38 and we'd only need a single import.